starport/pkg/events/events_test.go

```diff
@@ -0,0 +1,191 @@
+package events
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestBus_Send(t *testing.T) {
+	type args struct {
+		e Event
+	}
+	tests := []struct {
+		name string
+		b    Bus
+		args args
+	}{
+		{
+			name: "send status ongoing event",
+			b:    make(Bus),
+			args: args{Event{
+				status:      StatusOngoing,
+				Description: "description",
+			}},
+		},
+		{
+			name: "send status done event",
+			b:    make(Bus),
+			args: args{Event{
+				status:      StatusDone,
+				Description: "description",
+			}},
+		},
+		{
+			name: "send event on nil bus",
+			b:    nil,
+			args: args{Event{
+				status:      StatusDone,
+				Description: "description",
+			}},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			go func() {
+				tt.b.Send(tt.args.e)
+			}()
+			if tt.b != nil {
+				event := <-tt.b
+				if !reflect.DeepEqual(event, tt.args.e) {
+					t.Errorf("event = %v, want %v", event, tt.args.e)
+				}
+			}
+			tt.b.Shutdown()
+		})
+	}
+}
+
+func TestBus_Shutdown(t *testing.T) {
```

Comment:
After calling shutdown, we should actually check if the event can be received or not. Otherwise this test won't be too effective.